### PR TITLE
SW-2488 and SW-2487 Decrease size of “x” for mobile nav to match menu icon and fix disabled textfield active style

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2022-12-07 at 11 24 55

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "2.0.27",
+  "version": "2.0.29",
```
Comment (karthikbtf) [inline package.json:3]: should this be 28?

**Review by karthikbtf — COMMENTED** [review]
should we think about bunching multiple web-components fixes into one version?

File: src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -14,9 +14,9 @@ export interface Props {
 const useStyles = makeStyles(() => ({
   icon: {
     fill: '#708284',
```
Comment (karthikbtf) [inline src/components/Navbar/Navbar.tsx:16]: should this fill be using a token color? (unrelated to your PR but still)

File: package.json (hunk 1 above)
Comment (constanzauanini) [reply package.json:3]: I set 28 to the other PR, that's why this is 29

Comment (karthikbtf) [reply package.json:3]: just saw that, can we merge the 2 fixes in one PR?

File: src/components/Navbar/Navbar.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/Navbar/Navbar.tsx:16]: Good call out.. no we need to get new token value from @chudo76 perhaps.
We can follow up separately maybe.

**Review by karthikbtf — APPROVED** [review]

File: src/components/Navbar/Navbar.tsx (hunk 2 above)
Comment (chudo76) [reply src/components/Navbar/Navbar.tsx:16]: Token for " x " should be `tw-clr-icn-secondary`. For Kaloko, that would map to `#7F775B`.

Comment (constanzauanini) [reply src/components/Navbar/Navbar.tsx:16]: Thanks @chudo76 !

**Pull request merged**